Sources/Vapor/Routing/Parameters+Require.swift

```diff
@@ -0,0 +1,31 @@
+extension Parameters {
+    /// Grabs the named parameter from the parameter bag.
+    /// If the parameter does not exist, `Abort(.internalServerError)` is thrown.
+    /// If the parameter value cannot be converted to `String`, `Abort(.unprocessableEntity)` is thrown.
+    ///
+    /// - parameters:
+    ///     - name: The name of the parameter.
+    public func require(_ name: String) throws -> String {
+        return try self.require(name, as: String.self)
+    }
+
+    /// Grabs the named parameter from the parameter bag, casting it to a `LosslessStringConvertible` type.
+    /// If the parameter does not exist, `Abort(.internalServerError)` is thrown.
+    /// If the parameter value cannot be converted to the generic type, `Abort(.unprocessableEntity)` is thrown.
```

Comment:
Would rename this to the required type

```suggestion
    /// If the parameter value cannot be converted to the required type, `Abort(.unprocessableEntity)` is thrown.
```

Reply:
Done